folktables/exceptions.py

```diff
@@ -0,0 +1,17 @@
+class FileDownloadError(Exception):
```

Comment:
Would it make sense to have a base class for all of the folktable exceptions? Then user code could handle all of our exceptions with one `except` clause in case that's convenient in the user code (at least that's my limited understanding of Python's exception handling :-) ). It also wouldn't add much complexity here except an extra two lines for an empty class definition.

Reply:
I like the idea of creating a base class for all of the folktables exceptions. I'll go ahead and name it `FolktablesException`.

*I would, however, suggest we avoid trying to catch all exceptions at once within our code as it can lead to unintended consequences.*

Reply:
I'm not sure exactly what you mean by

> I would, however, suggest we avoid trying to catch all exceptions at once within our code as it can lead to unintended consequences.

Reply:
I'm just saying we should avoid using the following to handle errors:
```py
try:
    # do something
except Exception as e:
    # do something else
```
For instance if the user presses `<Control-C>` to terminate the running program, and does so while the program is executing stuff in the `try` block, then `SIGINT` signal will be caught in the `except` clause.

And we should also try to avoid handling all possible folktables exceptions types at once like so:
```py
try:
    # do something
except FolktablesException as e:
    # do something else
```

Reply:
Ah yes I agree with that, especially the first point.